packages/storybook/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "storybook",
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "private": true,
+  "backstage": {
+    "role": "frontend"
+  },
+  "sideEffects": false,
+  "scripts": {
+    "start": "storybook dev -p 6006",
+    "start:debug": "yarn start:storybook --debug-webpack",
+    "build": "storybook build",
+    "lint": "backstage-cli package lint",
+    "clean": "backstage-cli package clean"
+  },
+  "devDependencies": {
+    "@backstage/cli": "0.23.0",
+    "@storybook/addon-controls": "^7.5.3",
+    "@storybook/addon-essentials": "^7.5.3",
+    "@storybook/addon-interactions": "^7.5.3",
+    "@storybook/addon-links": "^7.5.3",
+    "@storybook/addon-onboarding": "^1.0.8",
+    "@storybook/blocks": "^7.5.3",
+    "@storybook/react": "^7.5.3",
+    "@storybook/react-webpack5": "^7.5.3",
+    "@storybook/testing-library": "^0.2.2",
+    "@testing-library/jest-dom": "^5.10.1",
+    "@types/react": "^17",
+    "@types/react-dom": "^17",
+    "eslint-plugin-storybook": "^0.6.15",
+    "prop-types": "^15.8.1",
+    "storybook-dark-mode": "^3.0.1"
+  },
+  "dependencies": {
+    "@swc/core": "^1.3.96",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "storybook": "^7.5.3",
+    "swc-loader": "^0.2.3"
+  },
+  "peerDependencies": {
+    "@backstage/core-app-api": "*",
+    "@backstage/core-plugin-api": "*",
+    "@backstage/test-utils": "*",
+    "@backstage/theme": "*"
+  },
+  "prettier": {
```

Comment:
I haven't seen any other package with custom prettier config. Do we need it?

Reply:
We don't, I'll remove it

Reply:
addressed in https://github.com/caponetto/backstage-plugins/pull/24/commits/f064fe4b27689f062d8d63ce370873ace0c26c6b